<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Sample name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="samplename">
     <property name="text">
      <string>2ndHarmonic_test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Folder:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="folder">
     <property name="text">
      <string>E:\Data\Temp\</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Maximum Angle of Rotation:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="max_angle">
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.900000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Delay:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="delay">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.350000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Minimum Magnet Current:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="min_curr">
     <property name="suffix">
      <string> A</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>40.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Maximum Magnet Current:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="max_curr">
     <property name="suffix">
      <string> A</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>40.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Current Step:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="curr_step">
     <property name="suffix">
      <string> A</string>
     </property>
     <property name="maximum">
      <double>40.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Lock-in 1:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Lockin1_use">
     <item>
      <property name="text">
       <string>1st Harmonic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2nd Harmonic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Lock-in 2:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Lockin2_use">
     <property name="currentText">
      <string>2nd Harmonic</string>
     </property>
     <item>
      <property name="text">
       <string>1st Harmonic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2nd Harmonic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
